lib/oplog/oplogEntry_test.go

```diff
@@ -1,4 +1,4 @@
-package oplog
+	package oplog
```

Comment:
nit: might be worth running a quick `go fmt` to clean up minor whitespace issues

Reply:
Good point -- just did, thanks!